.github/workflows/markdownlint.yml

```diff
@@ -0,0 +1,17 @@
+name: Markdown Lint Action
+on: push
+
+jobs:
+  build:
+    name: Markdown Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install markdown lint
+        run: sudo gem install mdl
```

Comment:
can we use the action here directly?
https://github.com/marketplace/actions/markdownlint-mdl-action

Reply:
Yup, but for simple stuff like this I tend to find just running the commands directly better. Rather than adding an extra layer, which I don't think adds any features? I think it just wraps the command-line tool. Using the commands directly makes it easier for people who want to reproduce locally. You don't need to know anything about Github Actions to figure out what is going on.

Reply:
Sounds reasonble, yea!